tests/unit/neuralnets/NeuralLeakyRectifiedLinearLayer_unittest.cc

```diff
@@ -30,75 +30,61 @@
  *
  * Written (W) 2015 Sanuj Sharma
  */
-
```

Comment:
add your name if you like

Reply:
its fine - i didnt really add any logic just removed boilerplate